fastify4ts/src/utils/controllerFactory.ts

```diff
@@ -0,0 +1,66 @@
+import { routes, utils } from '@contrast/test-bench-utils';
+import { forEach } from 'lodash';
+import { FastifyInstance, FastifyRequest, FastifyReply } from 'fastify';
+
+interface defaultRespond {
+  (result: unknown, request: FastifyRequest, reply: FastifyReply): void;
+}
+interface FactoryOptions {
+  locals?: Record<string, any>, respond?: defaultRespond
+}
+
+/**
+ * Custom response functions allow you to change the functionality or return
+ * value of a sink endpoint.
+ */
+const defaultRespond = (result: unknown, request: FastifyRequest, reply: FastifyReply) => {
```

Comment:
You could save yourself a few characters by def
```suggestion
const defaultRespond: Respond = (result, request, reply) => {
```